core/src/main/java/tachyon/client/RemoteBlockReader.java

```diff
@@ -39,5 +40,8 @@ public static RemoteBlockReader createRemoteBlockReader(TachyonConf conf) {
   }
 
   public abstract ByteBuffer readRemoteBlock(String host, int port, long blockId, long offset,
-      long length) throws IOException;
+      long length, TachyonConf conf) throws IOException;
```

Comment:
Any way around passing the conf to each method? Are there cases where the same reader will get different configurations for different methods?